edgexfoundry/edgex-compose
# feat: Make UI part of base services (secure and non-secure)

closes #199

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  https://github.com/edgexfoundry/edgex-docs/pull/630


## Testing Instructions

- Run any secure compose file
- Navigate to localhost:4000
- Verify API Gateway token is requested
- Get and enter a token
- Verify token is accepted and UI is functional
- Stop secure compose
- Run any non-secure compose
- Verify API Gateway token **IS NOT** requested
- Verify UI is functional
